app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -203,18 +219,37 @@ def kill_middleware_domain_server(ems_ref)
       run_generic_operation(:Kill, ems_ref)
     end
 
-    def shutdown_middleware_server(ems_ref, params)
+    # server group ops
+    def start_middleware_server_group(ems_ref)
+      # blocking: boolean
```

Comment:
These are the optional parameters the operation can take, I can remove that if it's confusing. Or comment better.